src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -119,6 +120,14 @@
     boolean isChatColorsEnabled();
 
     /**
+     * Simulates a chat message from a player.
+     *
+     * @param message The message to send
+     * @return The event thrown from the message
+     */
+    MessageChannelEvent.Chat chat(Text message);
```

Comment:
There should be a hint whether:
* This actually sends messages to someone or not
* Whether the event is already processed or not
* Whether you can send commands